package.json

```diff
@@ -39,7 +39,7 @@
     "jws": "^3.2.2",
     "lodash": "^4.17.21",
     "ms": "^2.1.1",
-    "semver": "^7.3.8"
+    "semver": "^7.5.2"
```

Comment:
```suggestion
    "semver": "^7.5.4"
```

Reply:
There's a few bugfixes from [7.5.3](https://github.com/npm/node-semver/releases/tag/v7.5.3) and [7.5.4](https://github.com/npm/node-semver/releases/tag/v7.5.4).